### PR TITLE
Added missing Semicolons

### PR DESCRIPTION
**Pull request by ErikThiart**

### CODE REVIEW

**Comment by ppelgrims** [comment]
The problem which caused the tests to fail is fixed. You can build again by closing and reopening this PR, rebasing and pushing, or by empty committing to this branch.

File: readme.md [hunk 1]
```diff
@@ -755,9 +755,9 @@ Fugiat non in itaque sunt nobis totam. Sed nesciunt est deleniti cumque alias. R
 <?php
 
 echo $faker->idNumber;      // ID number
-echo $faker->nationalIdNumber // Citizen ID number
-echo $faker->foreignerIdNumber // Foreigner ID number
-echo $faker->companyIdNumber // Company ID number
+echo $faker->nationalIdNumber; // Citizen ID number
+echo $faker->foreignerIdNumber; // Foreigner ID number
+echo $faker->companyIdNumber; // Company ID number
```
Comment (fzaninotto) [inline readme.md]: How about line 768? Did you check all the examples?

Comment (ErikThiart) [reply readme.md]: Oops, I missed 768, I added that one now and a few others too, I went through the whole readme file again, should be all now.

File: readme.md [hunk 2]
```diff
@@ -562,29 +562,29 @@ $faker = Faker\Factory::create();
 <?xml version="1.0" encoding="UTF-8"?>
 <contacts>
 <?php for ($i=0; $i < 10; $i++): ?>
-  <contact firstName="<?php echo $faker->firstName ?>" lastName="<?php echo $faker->lastName ?>" email="<?php echo $faker->email ?>">
-    <phone number="<?php echo $faker->phoneNumber ?>"/>
+  <contact firstName="<?php echo $faker->firstName; ?>" lastName="<?php echo $faker->lastName; ?>" email="<?php echo $faker->email; ?>">
```
Comment (fzaninotto) [inline readme.md]: no, semicolons are not required in this case, and they make the code less readable

**Comment by pimjansen** [comment]
@ErikThiart any update on this PR? Could you also rebase it to resolve the conflicts?

**Comment by pimjansen** [comment]
Going to close this issue as stale, please feel free to reopen if required

**Pull request closed without merging**